autoload/cmake/buildsys.vim

```diff
@@ -461,6 +444,7 @@ function! s:buildsys.Generate(clean, argstring) abort
     let l:run_options.callbacks_err = [function('s:RefreshConfigs')]
     let l:run_options.autocmds_pre = ['CMakeGeneratePre']
     call s:terminal.Run(l:command, 'GENERATE', l:run_options)
+    call s:fileapi.Parse(l:optdict.build_dir)
```

Comment:
Is this call to `Parse()` necessary? We will parse anyway in the `s:RefreshTargets()` callback, won't we?

Reply:
Indeed